src/google/protobuf/port_def.inc

```diff
@@ -851,7 +851,7 @@ static_assert(PROTOBUF_ABSL_MIN(20230125, 3),
 #endif  // PROTOBUF_TSAN
 
 #ifdef PROTOBUF_TSAN
-#define PROTOBUF_TSAN_DECLARE_MEMBER ::uint32_t _tsan_detect_race = 0
+#define PROTOBUF_TSAN_DECLARE_MEMBER ::uint32_t _tsan_detect_race = 0;
 #else
 #define PROTOBUF_TSAN_DECLARE_MEMBER
```

Comment:
instead of pushing this complexity into the macro and code base, can you simply add -Wpedantic to the set of warnings we suppress in port_def.inc?

Reply:
Probably